core/creators/themedui.js

```diff
@@ -253,12 +253,12 @@ CKEDITOR.replaceClass = 'ckeditor';
 	 *
 	 *		editor.resize( 900, 300 );
 	 *
-	 *		editor.resize( '100%', 450, true );
+	 *		editor.resize( '5in', 450, true );
```

Comment:
Using tabs for code listings is a deprecated method. Could you wrap it with markdown code listings i.e. backticks and programming language? E.g. see https://github.com/ckeditor/ckeditor4/blob/8d5d12dae4ee1fbd02c68db4c2a916957c204d87/plugins/autocomplete/plugin.js#L49-L101